### DISCUSSION TITLE
draw.io links do not work

### DISCUSSION THREAD

**Discussion by jan-runkel-zeitgleich** [requarks/wiki#3557] [Error / Bug Report]
**Describe the bug**
Draw.io support 2 Kinds of Links. Internal and External Links.
Both Links don't work with wiki.js 

**To Reproduce**
1. Go to any  markdown page
2. Click on 'insert Diagramm'
3. Click on 'insert page'
4. create link from page 1 to page 2
5. save and exit
6. save wikijs page
7. klick the link in the diagramm and see what happend.

**Host Info (please complete the following information):**
 - Wiki.js version: 2.5.144

**Comment by juliacherry12** [comment]
Hi, we have the same problem. Is there a solution?

**Reply by NGPixel** [reply]
Please [read the pinned issue](https://github.com/requarks/wiki/issues/6044) and avoid creating issues from discussions.

**Reply by juliacherry12** [reply]
This discussion has been unanswered since 2020.
And all previous ones on this subject have been closed by the bot due to inaction.
I would like to pay attention to it! And get an answer on what to do about the problem.
___
In our case, the links stop working because of the large number of objects in the draw.io diagram. This makes it impossible to create large diagrams with description links.
Wiki.js: v.2.5.295
NodeJS: v14.17.2

**Reply by jake-pfannenstiel** [reply]
I agree that this needs to be addressed. Links and mathematical typesets do not display up in the wiki.page
